go/tools/filter_tags/filter_tags.go

```diff
@@ -50,6 +51,7 @@ func filterFilenames(bctx build.Context, inputs []string) ([]string, error) {
 
 func main() {
 	cgo := flag.Bool("cgo", false, "Sets whether cgo-using files are allowed to pass the filter.")
+	quiet := flag.Bool("quiet", false, "Don't print filenames. Return code will be 0 if any files pass the filter.")
```

Comment:
just realized that all of these are local to main, which I guess is fine, but I am used to flags being in the main file's 'var' section, for clarity of what flags a binary supports.